bbc/simorgh
# WSTEAM1-456 - Live steam Post images

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-456

Overall changes
======
- Displays images on the Live page post stream
- Refactors the `ArticleFigure` component to be more succinct, removing stuff like the Grid wrappers to make it more usable on other page types
- Removes old `containers/Image` and `containers/Copyright` components. Copyright has been uplifted to `components`

Code changes
======
- Moves `Copyright` component to `components` directory. Removes old component and Psammead component
- Removes `containers/Image` and `containers/ArticleFigure` components in favour of new `ImageWithCaption` component, which combines both functionality together
- Updates the base `Image` component to allow `children` to be rendered into it
- Updates `ArticlePage`, `StoryPage`, `PhotoGalleryPage`, `MediaAssetPage` and `MediaArticlePage` pages to use new `ImageWithCaption` component
- Adds images to the Live stream posts

Testing
======
1. Change directory to `ws-nextjs-app`
2. Start the app with `yarn dev`
3. Visit http://localhost:7081/pidgin/new_live/c7p765ynk9qt
4. Confirm that images are now rendered on the page
5. Confirm that the images have the correct alt text (if supplied in the data)
6. Confirm that the images have captions (if supplied in the data)
7. **Note**: Overall styling will be changed later once the Post element is fully designed. This is just to display images on the posts.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
